packages/block-editor/src/components/grid/style.scss

```diff
@@ -105,7 +105,6 @@
 	.block-editor-grid-item-mover-button {
 		width: $block-toolbar-height * 0.5;
 		min-width: 0 !important; // overrides default button width.
-		overflow: hidden;
```

Comment:
To fix incorrect focus outline

![image](https://github.com/user-attachments/assets/195910f2-de24-42ef-8210-a8b862138cb5)